### PR TITLE
car parts blink red, or orange

### PR DESCRIPTION
**Pull request by JenniferJa**

### CODE REVIEW

**Comment by ideen1** [comment]
Remember to  rebase this change on top of main  before submitting 💯

File: app/src/renderer/code/components/molecules/HeroMolecules/CarGraphicComponent.tsx
Comment (CalebBourb) [inline app/src/renderer/code/components/molecules/HeroMolecules/CarGraphicComponent.tsx:1]: It's best practice to add a line of whitespace at the end of the file

**Review by alexwhelan12 — APPROVED** [review]
Looks good to me!

File: app/src/renderer/code/components/molecules/HeroMolecules/CarMolecules/CarModelComponent.tsx
Comment (CalebBourb) [inline app/src/renderer/code/components/molecules/HeroMolecules/CarMolecules/CarModelComponent.tsx:1]: same as before, add a line to the end of the file.

File: app/src/renderer/code/components/molecules/HeroMolecules/CarGraphicComponent.tsx [hunk 1]
```diff
@@ -1,14 +1,74 @@
-import React, { useState } from "react";
```
Comment (CalebBourb) [inline app/src/renderer/code/components/molecules/HeroMolecules/CarGraphicComponent.tsx:1]: It's best practice to add a line of whitespace at the end of the file

File: app/src/renderer/code/components/molecules/HeroMolecules/CarMolecules/CarModelComponent.tsx [hunk 2]
```diff
@@ -5,6 +5,8 @@ import { GLTF } from "three-stdlib";
 
 import { useGLTF } from "@react-three/drei";
 
+import { IndicationLocations, IndicationStates } from "../CarGraphicComponent";
```
Comment (CalebBourb) [inline app/src/renderer/code/components/molecules/HeroMolecules/CarMolecules/CarModelComponent.tsx:8]: same as before, add a line to the end of the file.

**Review by CalebBourb — CHANGES_REQUESTED** [review]
Need blank lines at the end of your files for best practices

**Review by IshaHaider — COMMENTED** [review]
reviewed. Looks good!

**Review by IshaHaider — APPROVED** [review]
LGTM

**Review by brian-ngyn — APPROVED** [review]

**Pull request merged**